### PR TITLE
Add drizzle basic drizzle schema and commands

### PR DESCRIPTION
**Pull request by ninest**
```bash
cd server
npm install

npm run db:push  # push the current schema to a local sqlite database

npm run db:studio  # open drizzle studio at https://local.drizzle.studio
```

I added a basic schema for a user. Try opening drizzle studio to confirm that the user table has been created

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -104,3 +104,6 @@ dist
 .tern-port
 
 server/node_modules/
+
+.env
```
Comment (ninest) [inline .gitignore:108]: we could do .env.local but that's gonna be more setup with dotenv

File: .gitignore [hunk 2]
```diff
@@ -104,3 +104,6 @@ dist
...
+local.db
```
Comment (ninest) [inline .gitignore]: just named my database local.db but we can probably do *.db

Comment (huang0h) [reply .gitignore]: let's do *.db so we can name our database whatever we want locally

Comment (ninest) [reply .gitignore]: did it

**Review by huang0h — CHANGES_REQUESTED** [review]
1 change, otherwise lgtm!

**Review by huang0h — APPROVED** [review]
:shipit:

**Review by arjbhat — APPROVED** [review]

**Pull request merged**